folly/Traits.h

```diff
@@ -500,6 +500,9 @@ template <class... Ts>
 struct StrictConjunction
     : std::is_same<Bools<Ts::value...>, Bools<(Ts::value || true)...>> {};
 
+template <template <class...> class Trait, class... Ts>
```

Comment:
Naming - `Trait` is odd - don't know what that's supposed to mean. Just `F` (for `Function`) would be fine.

Also why `template <class...> class` v.s. `template <class> class`?

Reply:
Both the name (`Trait`) and the signature with the template template just came from `Expected.h` as is. I suspect the intention was that our existing call sites use standard type traits as the metafunction, so naming the metafunction as `Trait` isn't too far from wrong, but this type trait is more general than that.

I agree that since this is a simple metafunction accepting only one type, we should name it as such (`F`) and make it so the template template parameter is just of one type for the class template.

I've made those changes as requested 👍